ChangeLog.md

```diff
@@ -4,6 +4,7 @@ QIIME 1.9.1-dev (changes since 1.9.1 go here)
 Bug fixes
 ---------
 
+* **Critical**: Fix incorrect list of taxa in ``compute_taxonomy_ratios.py``. **This was a serious bug that was encountered when users would call ``compute_taxonomy_ratios.py`` using the MD-index, custom ratios did not suffer from this bug. Any computations of the MD-index previosuly generated with that command should be re-run.**.
```

Comment:
thanks, fixed!

On (Dec-09-15|13:56), mortonjt wrote:
> > @@ -4,6 +4,7 @@ QIIME 1.9.1-dev (changes since 1.9.1 go here)
> >  Bug fixes
> >  ---------
> >  
> > +* **Critical**: Fix incorrect list of taxa in ``compute_taxonomy_ratios.py``. **This was a serious bug that was encountered when users would call ``compute_taxonomy_ratios.py`` using the MD-index, custom ratios did not suffer from this bug. Any computations of the MD-index previosuly generated with that command should be re-run.**.
> 
> previosuly -> previously
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiime/pull/2115/files#r47158218